### ISSUE TITLE
[CVE-2021-23017] Upgrade NGINX version

### ISSUE THREAD

**Issue by pjenicot** [jitsi/docker-jitsi-meet#1066]
Hi,

The current NGINX version in the Jitsi Web image [stable-5870] is :
```
nginx version: nginx / 1.14.2
built with OpenSSL 1.1.1d 10 Sep 2019
TLS SNI support enabled
```

Following the publication of the security alert [CVE-2021-23017](https://www.nginx.com/blog/updating-nginx-dns-resolver-vulnerability-cve-2021-23017/), could you update the version of NGINX to 1.20.1 or higher?

Thank you

**Comment by prayagsingh** [comment]
@saghul Is it possible to update the Nginx version? 
AFAIK Nginx is installed based on the Debian version we are using.

**Comment by saghul** [comment]
Is a Debian package available already in the Buster repos? If so yeah we can make a new release with an update.

**Comment by saghul** [comment]
Looks like it is indeed: https://metadata.ftp-master.debian.org/changelogs//main/n/nginx/nginx_1.14.2-2+deb10u4_changelog

**Comment by prayagsingh** [comment]
@saghul Current Nginx version in `stable-5870` or `latest` is `1.14.2-2+deb10u3`. I tried `unstable` and it is using nginx `1.14.2-2+deb10u4`.

**Comment by saghul** [comment]
Yep a new stable release will go out soon. The Debian version includes a back port of the fix to 1.14.

**Comment by saghul** [comment]
https://github.com/jitsi/docker-jitsi-meet/releases/tag/stable-5963 is out.
